src/core/server/base.js

```diff
@@ -83,13 +84,26 @@ export default function(routes, createStore) {
         );
 
         const componentHTML = renderToString(InitialComponent);
+
         const assets = webpackIsomorphicTools.assets();
-        const styles = Object.keys(assets.styles).map((style) =>
-          `<link href="${assets.styles[style]}" rel="stylesheet" type="text/css" />`
-        ).join('\n');
-        const script = Object.keys(assets.javascript).map((js) =>
-          `<script src="${assets.javascript[js]}"></script>`
-        ).join('\n');
+
+        // Get SRI for production only.
+        const sri = (ENV === 'production') ? JSON.parse(
+          fs.readFileSync(path.join(config.get('basePath'), 'sri.json'))
+        ) : null;
+
+        const styles = Object.keys(assets.styles).map((style) => {
+          const cssSRI = sri && ` integrity="${sri[path.basename(assets.styles[style])]}"
```

Comment:
This feels like it's getting a little hairy but I can't decide if it is better in JSX. If this is going to get more attributes added I'd say we should use JSX (or if we convert `HTML` to JSX) but I don't know what else we could add, really. I typed it up to see what it would look like so here it is. 

```jsx
const props = {
  rel: 'stylesheet',
  href: assets.styles[style],
  type: 'text/css',
};
if (sri) {
  props.integrity = sri[path.basename(props.href)];
  props.crossorigin = 'anonymous';
}
return renderToString(<link {...props} />);